src/test/abilities/synchronize.test.ts

```diff
@@ -93,17 +93,5 @@ describe("Abilities - Synchronize", () => {
     expect(game.scene.getPlayerPokemon()!.status?.effect).toBeUndefined();
     expect(game.scene.getEnemyPokemon()!.status?.effect).toBe(StatusEffect.PARALYSIS);
     expect(game.phaseInterceptor.log).toContain("ShowAbilityPhase");
-  });
-
-  it("should activate with Psycho Shift after the move clears the status", async () => {
-    game.override.statusEffect(StatusEffect.PARALYSIS);
-    await game.classicMode.startBattle();
-
-    game.move.select(Moves.PSYCHO_SHIFT);
-    await game.phaseInterceptor.to("BerryPhase");
-
-    expect(game.scene.getPlayerPokemon()!.status?.effect).toBe(StatusEffect.PARALYSIS); // keeping old gen < V impl for now since it's buggy otherwise
-    expect(game.scene.getEnemyPokemon()!.status?.effect).toBe(StatusEffect.PARALYSIS);
-    expect(game.phaseInterceptor.log).toContain("ShowAbilityPhase");
-  });
+  }, 20000);
```

Comment:
```suggestion
  });
```
Can you go through and remove the leftover `, 20000` from the test file too while you're there please?